xbmc/guilib/GUIFontTTFDX.h

```diff
@@ -47,20 +47,23 @@ class CGUIFontTTFDX : public CGUIFontTTF, public ID3DResource
 
 protected:
   std::unique_ptr<CTexture> ReallocTexture(unsigned int& newHeight) override;
-  bool CopyCharToTexture(FT_BitmapGlyph bitGlyph, unsigned int x1, unsigned int y1, unsigned int x2, unsigned int y2) override;
+  bool CopyCharToTexture(FT_BitmapGlyph bitGlyph,
+                         unsigned int x1,
+                         unsigned int y1,
+                         unsigned int x2,
+                         unsigned int y2) override;
   void DeleteHardwareTexture() override;
 
 private:
   bool UpdateDynamicVertexBuffer(const SVertex* pSysMem, unsigned int count);
   static void AddReference(CGUIFontTTFDX* font, CD3DBuffer* pBuffer);
   static void ClearReference(CGUIFontTTFDX* font, CD3DBuffer* pBuffer);
 
-  unsigned m_vertexWidth;
-  std::unique_ptr<CD3DTexture> m_speedupTexture; // extra texture to speed up reallocations
-  Microsoft::WRL::ComPtr<ID3D11Buffer> m_vertexBuffer;
+  unsigned m_vertexWidth{0};
+  std::unique_ptr<CD3DTexture> m_speedupTexture{nullptr}; // extra texture to speed up reallocations
```

Comment:
`std::unique_ptr` no need to initialize with nullptr, it's already done by default